notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb

```diff
@@ -0,0 +1,7314 @@
+{
```

Comment:
<p>use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>